components/bl-list-component/src/index.js

```diff
@@ -0,0 +1,28 @@
+import { List } from "./list";
+
+const { cn } = BackendlessUI.CSSUtils;
+
+export default function ListComponent({ component, eventHandlers }) {
+  const { classList, width, backgroundColor, type, listItems, color, fontSize } = component;
+
+  const onClickHandler = (e, key) => {
+    e.stopPropagation();
+    eventHandlers.onClickListItem({ item: key });
+  };
+
+  const props = {
+    type: type && type === "ol" ? "ol" : "ul",
+    color: color,
+    fontSize: fontSize,
+    onClickHandler: onClickHandler,
+  };
+
+  return (
+    <div
+      className={cn("bl-customComponent-list", classList)}
+      style={{ width: width, backgroundColor: backgroundColor }}
```

Comment:
`{ width: width, backgroundColor: backgroundColor }` is same as `{ width, backgroundColor }`